src/scriptnum/scriptnum.cairo

```diff
@@ -95,4 +98,16 @@ pub mod ScriptNum {
         };
         value.try_into().unwrap()
     }
+
+    pub fn int_32(mut n: i64) -> i64 {
```

Comment:
`int_32` should take a `i64` as its argument and return an `i32` just like the reference implementation here : https://github.com/btcsuite/btcd/blob/b161cd6a199b4e35acec66afc5aad221f05fe1e3/txscript/scriptnum.go#L160